### PR TITLE
media-gfx/inkscape: bump to 1.1 & sync live

### PR DESCRIPTION
**Pull request by arkamar**
This PR synces live ebuild with upstream and bunps inkscape to 1.1, where I also backported patch from upstream in order to support musl.

I enabled testing in live ebuild, where it works for me, but it is still restricted for 1.1 because few tests fail and it would be necessary to backport multiple quite big commits to make it work, which didn't make sense to me.

I am not sure if https://bugs.gentoo.org/show_bug.cgi?id=776265 should be closed as well.

### CODE REVIEW

**Comment by arkamar** [comment]
FP where `~riscv` is restored because it is present in `1.0.2-r2`, but I removed `~hppa` because `app-text/gspell` dependency does not have it as well.

**Comment by arkamar** [comment]
FP with dropped `~ia64` keyword bacause of the same reason as above, `app-text/gspell` dependency does not have it.

File: media-gfx/inkscape/inkscape-1.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,162 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
```
Comment (mikhailkoliada) [inline media-gfx/inkscape/inkscape-1.1.ebuild:6]: Do you really need an interval for only 2 python versions here?

**Review by mikhailkoliada — CHANGES_REQUESTED** [review]

File: media-gfx/inkscape/inkscape-1.1.ebuild (hunk 1 above)
Comment (arkamar) [reply media-gfx/inkscape/inkscape-1.1.ebuild:6]: No, I don't need it. This used more often than `{8,9}` in `::gentoo`.

Comment (arkamar) [reply media-gfx/inkscape/inkscape-1.1.ebuild:6]: Anyway, do you want me to change it?

Comment (arkamar) [reply media-gfx/inkscape/inkscape-1.1.ebuild]: Changed to `{8,9}` in recent FP.

**Pull request closed without merging**